jwst/model_blender/__init__.py

```diff
@@ -29,30 +29,3 @@
 
 
 __version__ = '0.9.3'
```

Comment:
We should also get rid of this version assignment. We're no longer tracking version at the level of individual modules. They got removed from most modules a while back, but a few instances were missed (like this one).

Reply:
I can open another PR for it. There are still several:

```python
# $ grep -ir '__version__ =' jwst/
jwst/lib/__init__.py:__version__ = '0.9.3'
jwst/skymatch/region.py:__version__ = '0.9.3'
jwst/skymatch/skystatistics.py:__version__ = '0.9.3'
jwst/tests/__init__.py:__version__ = '0.9.3'
jwst/model_blender/__init__.py:__version__ = '0.9.3'
jwst/csv_tools/__init__.py:__version__ = '0.9.3'
jwst/__init__.py:    __version__ = 'dev'
jwst/jwpsf/__init__.py:__version__ = '0.9.3'
jwst/extract_1d/extract1d.py:__version__ = '0.9.3'
jwst/cal_ver_steps.py:__version__ = "0.9.3"
jwst/exp_to_source/__init__.py:__version__ = '0.9.3'
jwst/extern/pyparsing.py:__version__ = "2.3.1" # not touching
jwst/extern/configobj/validate.py:__version__ = '1.0.1' # not touching
jwst/ramp_fitting/__init__.py:__version__ = '0.9.3'
```

Reply:
There's even an issue for that! #2940